Aztec/Classes/NSAttributedString/Styles/UnsupportedHTML.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+
+// MARK: - UnsupportedHTML NSAttributedString Attribute Name
+//
+let UnsupportedHTMLAttributeName = "UnsupportedHTMLAttributeName"
```

Comment:
Since this is part of the `UnsupportedHTML` logic... why not make it a static constant named `UnsupportedHTML.attributeName`?